examples/player/simid_player.js

```diff
@@ -197,33 +220,100 @@ class SimidPlayer {
   }
 
   /**
-   * Returns the dimensions of an element within the player div.
+   * Returns the full dimensions of an element within the player div.
+   * These dimensions will equal the full size of the player.
```

Comment:
This description doesn't make sense to me.  Is it getting the dimensions of the element within the player div, or the dimensions of the entire player div?  Depending on the element, those could be different values

Reply:
made comment more clear